go/vt/wrangler/migrater.go

```diff
@@ -239,7 +239,16 @@ func (wr *Wrangler) buildMigrater(ctx context.Context, targetKeyspace, workflow
 	if mi.sourceKeyspace != mi.targetKeyspace {
 		mi.migrationType = binlogdatapb.MigrationType_TABLES
 	} else {
+		// TODO(sougou): for shard migration, validate that source and target combined
```

Comment:
is this in a forthcoming PR? wondering why a TODO rather than implement it in this one.

Reply:
Yeah. I've added it to the list.